### PR TITLE
feature: add support for canboatjs to be able to send to the YDWG02

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: packages/streams/tcp.js [hunk 1]
```diff
@@ -42,6 +42,15 @@ function TcpStream (options) {
 require('util').inherits(TcpStream, Transform)
 
 TcpStream.prototype.pipe = function (pipeTo) {
+  if ( this.options.outEvent ) {
+    const that = this
```
Comment (tkurki) [inline packages/streams/tcp.js:46]: If the tcp stream is not connected yet / is disconnected the event handler will throw. Only write if `that.tcpStream` is defined? And make sure you unset it if we lose the connection.

Comment (sbender9) [reply packages/streams/tcp.js:46]: That is checked below in the event handler. So I think I just need to unset it?

Comment (tkurki) [reply packages/streams/tcp.js:46]: 👍

**Comment by tkurki** [comment]
Now that `@signalk/streams` is a separate module the PR should probably include a version bump in the module's package.json. There is no dependency to the main server, so only streams needs publishing technically. But I guess it makes sense to bump the version in server's package.json as well, makes for a more sane server changelog. And for the PR build to show green we need to publish streams first...

**Pull request merged**